templates/bot/ts/notification-function-base/src/sdk/middleware.ts

```diff
@@ -0,0 +1,53 @@
+import { Activity, Middleware, TurnContext } from "botbuilder";
+import { ConversationReferenceStore } from "./store";
+
+export interface NotificationMiddlewareOptions {
+  conversationReferenceStore: ConversationReferenceStore;
+}
+
+enum ActivityType {
```

Comment:
With the discussions about enum and literal types in previous [PR](https://github.com/OfficeDev/TeamsFx/pull/4152) , I wonder if we can update it to be like `Unknown = 'unknown'`.

Reply:
Yes. The SDK folder is currently just to make the scaffold code compiles fine. Later they will be implemented in teamsfx-sdk. So they will be removed here. Can we improve that in sdk later?